dask/array/percentile.py

```diff
@@ -228,17 +233,17 @@ def merge_percentiles(finalq, qs, vals, interpolation="lower", Ns=None):
     #
     # >>> A = np.concatenate(map(np.array, map(zip, vals, counts)))
     # >>> A.sort(0, kind='mergesort')
```

Comment:
Nice!  You can probably update or remove the comment block above.  I hope it was helpful.  Your solution below looks good.  (I wish I could go back and look at the variations I tried and my benchmarks, but I think they are lost to time)

Sanity check: you have `cytoolz` installed in the benchmark environment, right?  Based on your timings, I think it must be.

Reply:
Thanks for the feedback! Looking at the comment above, should it be:

```
# Sort by calculated percentile values
```

rather than

```
# Sort by calculated percentile values, then number of observations.
```

?

And yes, I do have `cytoolz` installed.